Source/UI/NewUI/BillingForm/View/Field/BillingFormPhoneNumberText.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+import PhoneNumberKit
+
+final class BillingFormPhoneNumberText: UITextField {
+    var type: BillingFormCellType
+    /// Phone Number Kit
+    let phoneNumberKit = PhoneNumberKit()
+    var partialFormatter: PartialFormatter {
+        return PartialFormatter.init(phoneNumberKit: phoneNumberKit, defaultRegion: "GB", withPrefix: true)
```

Comment:
Formatters are usually quite an expensive operation so creating a new one every time is not the most efficient approach.

I would instead suggest having a lazy loaded formatter that gets created locally:
ie:
```
private lazy var partialFormatter: PartialFormatter = createPhoneNumberFormatter()

private func createPhoneNumberFormatter() -> PartialFormatter {
    PartialFormatter()
}
```

Additionally, do we need to use the provided arguments? I am not sure defaultRegion "GB" is a safe assumption, and the `convenience initialiser` from PartialFormatter already uses the same defaults as we do.